python/tvm/autotvm/task/relay_integration.py

```diff
@@ -153,15 +154,25 @@ def extract_from_multiple_program(funcs, params, ops, target, target_host=None,
 
         logger.disabled = old_state
 
+    # convert *topi op to template key* map to *task name to template key* map
+    task_keys = dict()
+    for op in list(template_keys.keys()):
```

Comment:
Do we need `list()`? It seems to me that `for op in template_key.keys()` is sufficient.